VCFv4.3.tex

```diff
@@ -253,7 +253,7 @@ \subsubsection{Pedigree field format}
 ##pedigreeDB=URL
```

Comment:
This is not [the line I'm interested in](https://github.com/samtools/hts-specs/pull/413/files#diff-6fd0045c2c7902d4181b2e8d315cc56dR249) but I can't comment on the other one. PED does not support more than 2 ancestors, do we want to do it in VCF? I have never seen this used so I don't think a lot of people will miss it if we drop it. That will make trivial to add some examples for trios.

Reply:
@pd3 @lbergelson what should we do about this? Dropping support for more than 2 ancestors would render some files incorrect, but as I said in my previous comment I have never seen that syntax being used.

Reply:
I don't think there would be a lot of pushback against dropping multiple ancestor lines because they seem pretty ambiguously defined at the moment and I haven't ever seen one used... That said, we should maybe not be making breaking changes to an existing spec?

Are they intended only for asexual ancestry where each is the parent of the next?  Or are they just an unsorted bag of ancestors that could represent any tree of parentage?  

i.e does `<ID=SampleID,Name_1=Ancestor_1,...,Name_N=Ancestor_N>`
imply
``` 
SampleID -> Ancestor_1 -> Ancestor 2 
```
or could it also mean
```
 Ancestor_1 <- SampleID -> Ancestor_2
```
I lean towards removing or deprecating it if we don't know exactly what it means and no one seems to be using it.

I'm also not clear on which of these are controlled vocabulary.  Is there a specific ontology of relationships that are allowed?  Are we allowed to specify something like `Sibling` in the case where we don't have parent in the vcf or is that handled with dummy trios that point to unique but not present parents ID's

Are you allowed to include only 1 parent or are trios required?  

I assume the example would address some of these questions.